README.md

```diff
@@ -29,11 +29,11 @@ For a full list of external dependencies please consult `package.json`.
 
 ### NPM
 
-`npm install nightfall-node-sdk`
+`npm install nightfall-js`
```

Comment:
@evanfuller @rssathe how do you guys feel about this name?

Reply:
works for me